create_stacksentinel_run_files.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+########################
+# Author: Sara Mirzaee
+#######################
+
+import os
+import subprocess
+import sys
+import glob
+
+import argparse
+from rsmas_logging import loglevel
+import messageRsmas
+
+from _processSteps import create_or_update_template, create_or_copy_dem
+from _process_utilities  import get_work_directory, get_project_name
+from _process_utilities  import _remove_directories, send_logger
+
+sys.path.insert(0, os.getenv('SENTINEL_STACK'))
+sys.path.insert(0, os.getenv('SQUEESAR'))
+
+logger  = send_logger()
+
+##############################################################################
+EXAMPLE = """example:
+  create_stacksentinel_run_files.py LombokSenAT156VV.template 
+"""
+
+def create_parser():
+    """ Creates command line argument parser object. """
+
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter, epilog=EXAMPLE)
+    parser.add_argument('-v', '--version', action='version', version='%(prog)s 0.1')
+    parser.add_argument('custom_template_file', nargs='?',
+                        help='custom template with option settings.\n')
+
+    return parser
+
+
+def command_line_parse(args):
+    """ Parses command line agurments into inps variable. """
+
+    parser = create_parser()
+    inps = parser.parse_args(args)
+    
+    return inps
+
+
+
+if __name__ == "__main__":
```

Comment:
Anything in a `if __name__ == "__main__":` should be moved to a `main` function. At some point, it would be great if we can call all of these scripts in two ways. For example, we could call `create_stacksentinel_run_files.py` as follows:

From the command line:
```
create_stacksentinel_run_files.py $TEMPLATE
```
And also from within a python program
```
import create_stacksentinel_run_files
create_stacksentinel_run_files.main(arguments)
```

Reply:
There are (at least) two parts needed to get this working:
1. Anything in `if __name__ == "__main__":` must be moved to a function
2. We need to figure out what to do with `argparse`. There are no command line arguments if you call a script from another function.